lesson1/src/stacks_queues/stackProblems.js

```diff
@@ -19,12 +19,18 @@ class StackProblems {
    */
   static balancedParenthesis(string) {
     let stack = new Stack();
```

Comment:
nice 👍 i think you have an optimization to do, if you found a ')' char but the stack was empty you can just return false, since closing a parenthesis with no matching opening one should only happen for non balanced parenthesis. That and you compared chars in 2 different ways ```char === '('``` and ```char === ")"``` but hey, we are on an untyped language so that might work ! ahhaha